src/containers/Map/MapScreen.tsx

```diff
@@ -39,9 +39,9 @@ const MapScreen = () => {
 	const [inventories, setInventories] = useState<MapMenuProps['items'] | null>(
 		null
 	);
-	// If markers are bots, these these contain the paths of each non-idle bot
-	// else, if markers are map ndoes, contains all of the possible paths
-	const [botPaths, setBotPaths] = useState<Location[][] | null>(null);
+
+	// Path between selected Bot and selected Location
+	const [paths, setPaths] = useState<Location[][] | null>(null);
```

Comment:
We should leave it in because we will probably use it later. Its used to draw lines on the map, and we want to keep that functionality (just temporaritly not being used)